src/Learning/KWData/KWRelationCreationRule.cpp

```diff
@@ -634,16 +631,16 @@ boolean KWDRRelationCreationRule::CheckOperandsCompleteness(const KWClass* kwcOw
 							    sourceAttribute->GetClass()->GetName() !=
 								targetAttribute->GetClass()->GetName())
 							{
-								AddError("In the \"" + kwcTargetClass->GetName() +
-									 "\" output dictionary, the " +
+								AddError("In the " + kwcTargetClass->GetName() +
```

Comment:
IMHO, it's still good to delimit the interpolated parts. For more conciseness, single quote `'` could use, without needing t escape it.

Reply:
Suite à quelques essais, ce choix (questionnable) a été fait:
- pour améliorer la lisibilité (dans les cas simples de noms de variables élémentaire)
- pour se conformer au reste des usages dans le code

Le problème est qu'il y a plus de 1700 messages (AddMessage, AddWarning, AddError, AddFatalError) dans le code, et que c'est un peu compliqué de changer l'usage partout de façon homogène.
Cet usage pourra être remis en question plus tard (dans une issue dédiée), si on le juge pertinent et prioritaire.